presto-main/src/main/java/com/facebook/presto/connector/informationSchema/InformationSchemaPageSourceProvider.java

```diff
@@ -320,6 +354,15 @@ private static QualifiedTablePrefix extractQualifiedTablePrefix(String catalogNa
         return new QualifiedTablePrefix(catalogName, schemaName, tableName);
     }
 
+    private static String extractGrantee(Session session, Map<String, NullableValue> filters)
+    {
+        Optional<String> grantee = getFilterColumn(filters, "grantee");
```

Comment:
I believe `getFilterColumn` will take care of non-equality filters. I tested for `>` and `<` conditions, and they work.